tests/main/interfaces-many-core-provided/task.yaml

```diff
@@ -39,6 +47,10 @@ prepare: |
     fi
 
 restore: |
+    if os.query is-trusty; then
+        echo "Ubuntu trusty is not supported, exiting..."
+        exit 0
+    fi
```

Comment:
my preference would be instead of skipping the whole test, to comment out the problematic plug when installing the snap on trusty

Reply:
something like this woudl accomplish that

```
cp -r "$TESTSLIB/snaps/test-snapd-policy-app-consumer" .
shm=$(grep -n shared-memory: test-snapd-policy-app-consumer/meta/snap.yaml|cut -d ':' -f1)
ed=()
n=0
for l in $shm; do
    ed[$n]="-e $l,+2d"
    n=$((n + 1))
done
sed -i "${ed[@]}" test-snapd-policy-app-consumer/meta/snap.yaml
```

Reply:
@pedronis good idea, thanks for the review, I just removed the plug and interface

Reply:
@mardy Also updated the interfaces-many-snap-provided test here.

Reply:
Ah, nice, I just pushed a commit with Samuele's suggestion, but your variant is indeed much simpler.